decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -101,7 +108,7 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
     
     func seedWordEntered(for wordIndex: Int, seedWord: String, moveToNextField: Bool) {
         self.userEnteredSeedWords[wordIndex] = seedWord
-        
+        self.lblEnterAllSeeds.isHidden = true
```

Comment:
- Add an empty line space after this
- If the view was visible and you hide it, there's a weird space that shows up. You need to do this:
```
// increase top spacing since warning label is now hidden so as to position button in center
        self.tableViewFooterTopSpacingConstraint.constant = 30
        UIView.animate(withDuration: 0.5) {
            self.tableViewFooter.layoutIfNeeded()
        }
```

Reply:
There is no weird space showing up